java-2016/homework-9/IOhomework/src/Testing.java

```diff
@@ -0,0 +1,146 @@
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("UK",expected.get(1).getCountryCode());
+    }
+
+    @Test
+    public void readCountryCodeFromValidFile2(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("CZ",expected.get(2).getCountryCode());
+    }
+
+    @Test
+    public void readSkiTimeFromValidFile1(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("30:10",expected.get(2).getMinuteSeconds());
+    }
+
+    @Test
+    public void readSkiTimeFromValidFile2(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("30:27",expected.get(0).getMinuteSeconds());
+    }
+
+    @Test
+    public void readShootingRangeFromValidFile1(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("xxxox",expected.get(0).getFirstShootingRange());
+    }
+
+    @Test
+    public void readShootingRangeFromValidFile2(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
+        List<Athlete> expected = read.getAthletes();
+
+        Assert.assertEquals("xxxxx",expected.get(2).getSecondShootingRange());
+    }
+
+    @Test
+    public void readShootingRangeFromValidFile3(){
+        File file = new File("C:" + File.separator + "Users" + File.separator + "Ovi" + File.separator + "Desktop" + File.separator + "Curs" + File.separator + "IOFiles" + File.separator + "Athletes.csv");
+        ReadFile read = new ReadFile();
+        read.readFile(file);
```

Comment:
This is not really useful. You basically have a bunch of tests to check the behavior of just one method. You could have checked all this in 2-3 tests. One to read all and check number, country, etc and one to check for invalid files.